_config.yml

```diff
@@ -12,6 +12,10 @@ collections:
     title: All Articles
     output: true
     permalink: "/:path/articles/"
+  posts:
```

Comment:
This should be `themes` not `posts`.

Reply:
Edited!

Confirming: (Articles) are post layouts. Themes/Events/Series/Themes are citing their default front-matter templates that I had made in Forestry?

Also, the front matter has to be updated as Tags/Keywords lists are made.